.github/workflows/test-suite.yml

```diff
@@ -10,6 +10,10 @@ on:
       provider_matrix:
         description: 'Previous MongoDB Atlas Provider version matrix for migration tests (JSON array)'
         default: '[""]' # "" for latest version
+      send_notification:
+        description: 'Send the Slack notification if any of the tests fail. Can be either `true` or `false`.'
+        type: boolean
+        default: true
```

Comment:
In case we trigger a test suite execution manually from GH UI, even if we leave this in true is wont run because notifications are sent when `github.event_name == 'schedule'`. Can this parameter be removed for the case of dispatch to avoid confusion?

Reply:
Makes sense! Removed

Reply:
I might have misunderstood. I removed `github.event_name == 'schedule'` from the condition to send the notification, but added the input `send_notification` in both workflow_call and workflow_displatch (defaulting to true in both) so that we can choose if we want to send it when running it manually. 
You meant to remove the input completely from `workflow_dispatch`?

Reply:
I see, yes I was thinking that for the case of dispatching an execution manually there would be no need of notifying in slack. Maybe we can just adjust the default to `false` for this case?

Reply:
Okay, I changed the default to false. Thanks!